BUILD.osx.md

```diff
@@ -3,37 +3,48 @@ Build instructions for Mac OS
 
 Environment Setup
 -----------------
+
 ### XCode
-Can be installed via the App Store. Once this is done, you may need to install the command line tooling too, to do this run:
+
+Can be installed via the App Store. Once this is done, you may need to install the command line tooling too, to do this
+run:
 
     xcode-select --install
 
-Ensure you have development Frameworks for at least OS X 10.8 installed, with the typical compiler toolchain and "git". Avoid the version of cmake supplied, we need a newer one (see later).
+Ensure you have development Frameworks for at least OS X 10.8 installed, with the typical compiler toolchain and "git".
+Avoid the version of cmake supplied, we need a newer one (see later).
 
 ### Qt6
+
 #### Option 1: Using Qt official sources
+
 Install the latest stable version of Qt6 LTS (6.2.4 at the moment): <http://www.qt.io/download-open-source/>.
 
-If it tells you that XCode 5.0.0 needs to be installed, go to XCode > Preferences > Locations and make a selection in the _Command Line Tools_ box.
+If it tells you that XCode 5.0.0 needs to be installed, go to XCode > Preferences > Locations and make a selection in
+the _Command Line Tools_ box.
 
 Add Qt6 to your PATH environment variable, adding to your `.bash_profile` file the following line:
 
     export PATH=$PATH:~/Qt/6.2.4/clang_64/bin
```

Comment:
This might be different based on architecture. On my ARM Mac, it's at `~/Qt/6.2.4/macos/bin`